src/components/tables/adoptionReportTable/index.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { AdoptionReportEntry } from '../../../containers/reports/ducks/types';
+import { Table } from 'antd';
+import { dateSorter, DESCEND_ORDER, renderSiteIdAsLink } from '../utils';
+
+const AdoptionReportTableColumns = [
```

Comment:
🏕️ We may want to bring this out into its own file for readability

Reply:
Used [JSX style API](https://ant.design/components/table/#components-table-demo-jsx) instead.